File: fc-chat/demo/part3/ch3/03-1-message-system-add-channel/src/main/java/net/prostars/messagesystem/dto/websocket/outbound/FetchUserInvitecodeResponse.java
package net.prostars.messagesystem.dto.websocket.outbound;

import net.prostars.messagesystem.constant.MessageType;
import net.prostars.messagesystem.dto.domain.InviteCode;

public class FetchUserInvitecodeResponse extends BaseMessage {

  private final InviteCode inviteCode;

  public FetchUserInvitecodeResponse(InviteCode inviteCode) {
    super(MessageType.FETCH_USER_INVITECODE_RESPONSE);
    this.inviteCode = inviteCode;
  }

  public InviteCode getInviteCode() {
    return inviteCode;
  }
}

File: fc-chat/demo/part3/with-push/src/main/kotlin/com/yscorp/withpush/messagesystem/dto/websocket/outbound/FetchUserInvitecodeResponse.kt
package com.yscorp.withpush.messagesystem.dto.websocket.outbound

import com.yscorp.withpush.messagesystem.dto.domain.InviteCode
import com.yscorp.withpush.messagesystem.constant.MessageType

class FetchUserInvitecodeResponse(val inviteCode: InviteCode) : BaseMessage(MessageType.FETCH_USER_INVITECODE_RESPONSE) {
}
